text/0010-burn-coretime-revenue.md

```diff
@@ -0,0 +1,33 @@
+# RFC-0010: Burn Coretime Revenue
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 19.07.2023                                                                   |
+| **Description** | Revenue from Coretime sales should be burned                                                                   |
+| **Authors**     |                   Jonas Gehrlein                                                                          |
+
+## Summary
+
+The Polkadot UC will generate revenue from the sale of available Coretime. The question then arises: how should we handle these revenues? Broadly, there are three potential paths – burning the revenue and thereby removing it from total issuance or divert it to the Treasury. This Request for Comment (RFC) presents arguments favoring burning as the preferred mechanism for handling revenues from Coretime sales.
```

Comment:
I can only see 2 paths discussed here: burning or put into treasury. What's the 3rd option?

Reply:
you are right, I'll fix it. Originally, I also thought about the possibility to divert revenues to validators. But after making the point about "Clear Incentives", this struck me as an option not even worth including.